src/lilURL.php

```diff
@@ -61,13 +69,46 @@ public function handleRedirect($id)
         if ($id != '' && $id != basename($_SERVER['PHP_SELF']) && $id != '?login') {
             $location = $this->getURL($id);
             if ($location != false) {
+                $this->trackHit($id);
```

Comment:
Can you explain the problem with the Async method more, I'm not sure I understand. My understanding, per the library's documentation, is that by using the async method it will simply post to the GA endpoint and not wait for a response from GA (think a HEAD request). Is that not what happens? While not perfect, I see that as good middle ground in reducing the chances of issues arising.